lib/segments/pr1.rb

```diff
@@ -0,0 +1,24 @@
+# encoding: UTF-8
+# via https://github.com/bbhoss/ruby-hl7/blob/master/lib/segments/pr1.rb
+class HL7::Message::Segment::PR1 < HL7::Message::Segment
+  add_field :set_id
+  add_field :coding_method
+  add_field :code
+  add_field :description
+  add_field :date_time
```

Comment:
**todo:** datetime fields should be formatted like we do everywhere else, with the help of this method: https://github.com/ruby-hl7/ruby-hl7/blob/85028d6c4fdbf88c9860cabd533557a39cc01d98/lib/segment.rb#L195